pkg/doc/verifiable/signaturetype.go

```diff
@@ -46,6 +46,16 @@ type signatureTypeDesc struct {
 	SupportedKeyTypes []kms.KeyType
 }
 
+// SDJWTSignatureTypes stores the list of SD-JWT supported SignatureType by VCS
+// nolint: gochecknoglobals
+var SDJWTSignatureTypes = []string{
+	EdDSA.Name(),
+	ES256K.Name(),
+	ES256.Name(),
+	ES384.Name(),
+	PS256.Name(),
+}
```

Comment:
Used with `verifier.WithIssuerSigningAlgorithms(vcsverifiable.SDJWTSignatureTypes))` to extend the list of supported algorithms.